src/code_1A5830.c

```diff
@@ -1,12 +1,51 @@
 #include "common.h"
 
-INCLUDE_ASM(s32, "code_1A5830", func_80276F50);
+void dispatch_event_actor(Actor* actor, Event event);
+
+s32 func_80276F50(Actor* actor) {
+    ActorPart* partIt = actor->partsTable;
+    s32 ret = FALSE;
+
+    while (partIt != NULL) {
+        if (partIt->eventFlags & 0xC0000) {
+            ret = TRUE;
+            break;
```

Comment:
`return TRUE` doesn't match? why did they write it like this lol

Reply:
It doesn't...yeah lol